packages/fuels/tests/wallets.rs

```diff
@@ -270,7 +270,8 @@ async fn send_transfer_transactions() -> Result<()> {
 
     // wallet_1 has now only one spent coin
     assert_eq!(wallet_1_spendable_resources.len(), 1);
-    assert_eq!(wallet_1_all_coins.len(), 1);
+    // But it has two coins (one spent, one unspent (the change))
+    assert_eq!(wallet_1_all_coins.len(), 2);
```

Comment:
The core stopped returning spent coins one or two versions back. So the correct num of coins here is 1 (also the test fails).